package.json

```diff
@@ -103,5 +104,12 @@
     "*.{css,scss,json,html,yaml,md,mdx}": [
       "prettier --write"
     ]
+  },
+  "prettier": {
```

Comment:
We use eslint-plugin-prettier to format things, running eslint on a file should take care of the formatting/js issues in one go.

Reply:
Yeah, was running into issues with format-on-save so added that in lol. This would also correspond to the `format` command too that runs on commit for the file types listed, right? Or are the settings shared somewhere for formatting those files?

Reply:
Also forgot to mention, it specifically was for .mdx files to be formatted with my editor.

Reply:
Let me know if it makes sense to have a shared prettier config either here or in `prettier.config.js` for this use-case (that can also be read by `eslint-plugin-prettier`) 👀